package.json

```diff
@@ -1,33 +1,45 @@
 {
-  "name": "@parity/dapp-addresses",
-  "description": "Parity default addressbook view",
-  "version": "1.99.99",
+  "name": "bond-dapp-addresses",
+  "version": "1.0.0",
+  "description": "address dapp using bonds",
```

Comment:
Keep original description - implementation doesn't make a difference, this is just teh next version of the same functionality.